discovery_engine_core/ai/ai/src/document.rs

```diff
@@ -30,14 +29,13 @@ impl From<Uuid> for DocumentId {
 
 /// Common document properties.
 pub trait Document {
+    type Id: std::fmt::Display + std::fmt::Debug + Eq + std::hash::Hash;
```

Comment:
🎨 
```suggestion
    type Id: std::fmt::Display + Debug + Eq + Hash;
```
+ adding `use std::{fmt::Debug, hash::Hash};` above